src/main/java/edu/uc/cech/agrawadv/eventorganizer/user/UserService.java

```diff
@@ -5,9 +5,10 @@
 import java.util.UUID;
 
 public class UserService {
+	
```

Comment:
Added spacing because the rest of your code has a one line above and below the code for readability.  ref :https://css-tricks.com/write-better-code-3-levels-code-consistency/